sources/packages/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -112,12 +120,66 @@ export class ApplicationController extends BaseController {
         `Application id ${applicationId} was not found.`,
       );
     }
+    const additionalFormData = {} as ApplicationFormData;
```

Comment:
Line 123 to 171 almost looks same between application.students and application.aest controller. Is is possible to introduce a controllerService and reuse the logic?

Reply:
please refresh and have a look. its not same

Reply:
Got it.

Reply:
I see that you have extracted the possible common logic inside controller service 👍

Reply:
I see that you have extracted the possible common logic inside controller service 👍